src/doom/p_switch.c

```diff
@@ -25,7 +25,7 @@
 #include "p_local.h"
 #include "i_swap.h" // [crispy] SHORT()
 #include "w_wad.h" // [crispy] W_CheckNumForName()
-#include "z_zone.h" // [crispy] Z_ChangeTag()
+#include "z_zone.h" // [crispy] purge tags
```

Comment:
Strictly speaking, we only include this for the `PU_STATIC` in `W_CacheLumpName("SWITCHES", PU_STATIC)`. So, `PU_STATIC` would be the right comment.